presidio-analyzer/presidio_analyzer/conf/default_recognizers.yaml

```diff
@@ -0,0 +1,140 @@
+supported_languages: 
+  - en
+
+recognizers:
+  # Recognizers listed here can either be loaded from the recognizers defined in code (type: predefined),
+  # or created based on the provided configuration (type: custom).
+  # For predefined:
+  # - If only a recognizer name is provided, a predefined recognizer with this name and default parameters will be loaded.
+  # - If a parameter isn't provided, the default one would be loaded.
+  # For custom:
+  # - See an example configuration here: https://github.com/microsoft/presidio/blob/main/presidio-analyzer/presidio_analyzer/conf/example_recognizers.yaml
+  # - Custom pattern recognizers with this configuration can be added to this file, with type: custom
+  #  For recognizers supporting more than one language, an instance of the recognizer for each language will be created.
+  # For example, see the CreditCardRecognizer definition below:
+  - name: CreditCardRecognizer
+    supported_languages:
+    - language: en
+      context: [credit, card, visa, mastercard, cc,
+      amex, discover, jcb, diners, maestro, instapayment]
+    - language: es
+      context: [tarjeta, credito, visa, mastercard, cc,
+      amex, discover, jcb, diners, maestro, instapayment]
+    - language: it
+    - language: pl
+    type: predefined
+
+  - name: UsBankRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: UsLicenseRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: UsItinRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+  
+  - name: UsPassportRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+  
+  - name: UsSsnRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: NhsRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: SgFinRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: AuAbnRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: AuAcnRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: AuTfnRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: AuMedicareRecognizer
+    supported_languages: 
+    - en
+    type: predefined
+
+  - name: InPanRecognizer
+    supported_languages: 
+    - en
```

Comment:
in?

Reply:
Yes, but one can change this pretty easily if needed